creating_and_sending_custom_greeting_cards/service.py

```diff
@@ -18,11 +20,11 @@ def create_and_send_greeting():
         occasion = json_body.get('occasion')
         recipient_email = json_body.get('recipient_email')
         email_subject = json_body.get('email_subject')
-        special_notes = json_body.get('special_notes') or ""
+        special_notes = json_body.get('special_notes', '')
 
         if occasion is None or recipient_email is None or email_subject is None:
             logging.error("Request is not properly formatted")
-            return
+            return "ERROR: Request is not properly formatted.", HTTP_400_BAD_REQUEST_ERROR
 
         with ThreadPoolExecutor() as executor:
```

Comment:
When the ThreadPoolExecutor() is used, it creates a separate pool of threads that are distinct from the threads created by Flask. The executor.submit() method will submit a new task to the ThreadPoolExecutor pool, and the pool will assign it to one of the threads in the pool to execute. Because the ThreadPoolExecutor creates its own pool of threads, this means that multiple requests coming in at the same time will result in the creation of multiple thread pools, each with their own set of threads.